invoice/transactions/payment_transaction.js

```diff
@@ -15,44 +15,38 @@ class PaymentTransaction extends TransferTransaction {
 	 }
 
 	applyAsset(store) {
-		super.applyAsset(store);
-		const errors = [];
 		const transaction = store.transaction.find(
 			transaction => transaction.id === this.asset.data
 		); // Find related invoice in transactions for invoiceID
 
-		if (transaction) {
-			if (this.amount.lt(transaction.asset.requestedAmount)) {
-				errors.push(
-					new TransactionError(
-						'Paid amount is lower than amount stated on invoice',
-						this.id,
-						'.amount',
-						transaction.requestedAmount,
-						'Expected amount to be equal or greated than `requestedAmount`',
-					)
-				);
-			}
-		} else {
-			errors.push(
-				new TransactionError(
+		if (!transaction) {
+            return [ new TransactionError(
```

Comment:
I prefer to have the errors array as it's easier to understand what this array is doing?